addons/l10n_cl/models/res_partner.py

```diff
@@ -26,7 +26,10 @@ class ResPartner(models.Model):
     def _format_vat_cl(self, values):
         identification_types = [self.env.ref('l10n_latam_base.it_vat').id, self.env.ref('l10n_cl.it_RUT').id,
                                 self.env.ref('l10n_cl.it_RUN').id]
-        partner_country_is_chile = values.get('country_id') == self.env.ref('base.cl')
+        partner_country_is_chile = (values.get('country_id') == self.env.ref('base.cl').id) or (
+                    values.get('l10n_latam_identification_type_id') and
+                    self.env['l10n_latam.identification.type'].browse(
+                        [values.get('l10n_latam_identification_type_id')]).country_id == self.env.ref('base.cl'))
         if (partner_country_is_chile or not values.get('country_id')) and \
```

Comment:
Can we remove the `not values.get('country_id')`?  Maybe we can put the values.get('l10n_latam_iden... in a variable (or even the browse record)

Reply:
@Danisan